account_invoice_split_invoice/wizard/account_invoice_split_wizard.py

```diff
@@ -9,32 +9,32 @@
 #
 ##############################################################################
 
-from openerp import api, fields, models
+from openerp import api, fields, models, netsvc, _
 from openerp import netsvc
 
-
+
 class split_invoice_wizard(models.TransientModel):
     """Wizard for selecting invoice lines to be moved
     onto a new invoice."""
     _name = 'account.invoice.split.wizard'
 
-    @api.model
-    def _get_invoice(self):
-        return self.env.context.get('active_id')
+    invoice_id = fields.Many2one(
+        'account.invoice', default=lambda self: self._get_invoice())
 
     invoice_line_ids = fields.Many2many(
         'account.invoice.line', 'account_invoice_line_2_splitwizard',
         string='Invoice lines')
+
+    @api.multi
+    def _get_invoice_id(self):
+        return self.env.context.get('active_id')
 
-    invoice_id = fields.Many2one(
-        'account.invoice', default=_get_invoice)
+    @api.model
+    def _get_invoice(self):
+        return self.env.context.get('active_id')
 
-    def _copy_invoice(self, old_invoice):
-        # Create new invoice
-        new_invoice = old_invoice.copy(
-            default={'date_invoice': old_invoice.date_invoice})
-        new_invoice.invoice_line.unlink()
-        return new_invoice
+        line_ids = [line.id for line in invoice.invoice_line]

```

Comment:
This line is never reachable since return is called before. It may be a bad merge during rebase...